drop-boxes/register-omero-metadata/register-omero.py

```diff
@@ -159,68 +236,119 @@ def process(transaction):
 
 	# Get the incoming path of the transaction
 	incomingPath = transaction.getIncoming().getAbsolutePath()
+	# Get the name of the incoming folder
+	folderName = transaction.getIncoming().getName()
+
+	print incomingPath
 
 	# 1. Initialize the image registration process
 	registrationProcess = irp.ImageRegistrationProcess(transaction)
+
+	print "started reg. process"
 	
 	# 2. We want to get the openBIS sample code from the incoming data
 	# This tells us to which biological sample the image data was aquired from.
 	project_code, sample_code = registrationProcess.fetchOpenBisSampleCode()
 
+	print project_code
+	print sample_code
+
+	#find specific sample
+	tissueSample = registrationProcess.searchOpenBisSample(sample_code)
+	space = tissueSample.getSpace()
+
+	print tissueSample
+	print space
 	# 3. We now request the associated omero dataset id for the openBIS sample code.
```

Comment:
Causes not informative log flooding. Can you please make that either more informative or remove it please.

Reply:
I removed the first line, and commented the others